vsts/pipelines/templates/_securityChecks.yml

```diff
@@ -18,6 +24,11 @@ steps:
     debugMode: false
   condition: always()
 
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-roslynanalyzers.RoslynAnalyzers@2
+  displayName: 'Run Roslyn Analyzers'
+  continueOnError: true
```

Comment:
I'm curious about this `continueOnError: true`. If anything is detected, wouldn't it will be better to let the developer know immediately by stopping the build? Aren't we going to miss some things if we continue by default?

Reply:
The perils of copy-paste 😀

Reply:
Haha, thanks for clarifying